spec/features/idv/in_person_spec.rb

```diff
@@ -411,4 +411,74 @@
       expect(page).to have_current_path(idv_doc_auth_welcome_step)
     end
   end
+
+  context 'transliteration'
```

Comment:
this doesn't throw an error, but it also doesn't get printed when running tests; we should follow convention and make it a block that wraps the new test below
```ruby
  context 'transliteration' do
    ...
  end
```

Reply:
I intended to use a block here. Thanks for pointing out the issue.